bimsb/packages/tainted.scm

```diff
@@ -177,7 +177,7 @@ deviation (SD) plots, coefficient of variation (CV) plots.")
                 "1mlihj6c6zkqsvrdqnz5wjz4annjr8yi6slv63bbcl4bdwdywcd6"))))
     (build-system gnu-build-system)
     (arguments
-     `(#:tests? #f ;; no check target
+     `(#:tests? #f ;; no check target; TODO: Run tutorial as test?
```

Comment:
As long as its non-interactive: sure.

Reply:
It is.

Reply:
I'll still need to take care of this before this PR is ready to merge.

Reply:
The latest push finally took care of this.